test-network-k8s/README.md

```diff
@@ -0,0 +1,75 @@
+# Kubernetes Test Network 
+
+This project re-establishes the Hyperledger [test-network](../test-network) as a _production deployment_ on Kubernetes.
+
+_Fabric, Ahoy!_
+
+
+### Objectives:
+
+- Provide **simple**, "one-click" actions for deploying production-grade Fabric networks on Kubernetes.
+- Provide a reference Kubernetes sample, illustrating community recommendations for CAs, MSP, and operational activities.  
+- Align dev, test, CI/CD, and production pipelines on a _common_, cloud-native runtime (Kubernetes).
+- Support a transition from Internal, [Docker daemon](link) to [External Chaincode](link) builders.
+- Support a transition to [Chaincode as a Service](link) deployments
+- Provide a K8s reference and supplement to Fabric [CA Operations](link) and [CA Deployment](link) guides.
+- Kube-agnostic : AWS, Azure, gcp, IKS, KIND, OCP, ... 
+
+
+## Prerequisites 
+
+- [Docker](https://www.docker.com)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [KIND](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+
+
+## Quickstart
+
+Create a Kubernetes cluster:
+```shell
+./network kind
+```
+
+Launch the test network, create a channel, and deploy the [basic-asset-transfer](../asset-transfer-basic) smart contract: 
+```shell
+./network up
+./network channel create
+./network chaincode deploy
+```
+
+Chaincode:
+```shell
+./network chaincode invoke '{"Args":["CreateAsset","1","blue","35","tom","1000"]}' 
+./network chaincode query '{"Args":["ReadAsset","1"]}'
+```
+
+Tear down the test network: 
+```shell
+./network down 
+```
+
+Tear down the Kubernetes cluster: 
+```shell
+./network unkind
+```
+
```

Comment:
What state is held if any... if you kind-unkind-kind  is there any state left?
What about kind-up-down-up?

Reply:
that's good... how is the registry cleanup though?  Primarily thinking disk space here.

(aside if it's not already can you copy that answer into the docs?